website/pages/docs/reference/cli/cloudquery_sync.md

```diff
@@ -38,7 +39,7 @@ cloudquery sync ./directory ./aws.yml ./pg.yml
       --log-format string        Logging format (json, text) (default "text")
       --log-level string         Logging level (default "info")
       --no-log-file              Disable logging to file
-      --telemetry-level string   Telemetry level (none, errors, stats, all) (default "all")
+      --telemetry-level string   Telemetry level (none, errors, stats, all) (default "none")
```

Comment:
Not sure why this changed 🤔

Reply:
Oh, it's affected by having `CQ_NO_TELEMETRY` set locally. We'll need to fix this

Reply:
Weird.

Reply:
It's actually fine I think. Yes, it is affected by `CQ_NO_TELEMETRY`, but that environment variable is considered deprecated and developers should be using `CQ_TELEMETRY_LEVEL=none` locally now. If you do that, the default remains `all`. Unfortunately I don't see a way to support the legacy `CQ_NO_TELEMETRY` and not have it affect the default value. :/

Reply:
It's a bit blunt but https://github.com/cloudquery/cloudquery/pull/3180 should help with this.
Not sure if there's a better way to do it via Cobra as I guess we always want to generate the docs without any env variables set